extension/src/store/actions.js

```diff
@@ -77,6 +77,31 @@ export default ({ apollo }) => {
     )
   }
 
+  const deleteAccount = ({ commit }, { address, password }) => {
```

Comment:
is this used?

Reply:
no, it is not. It is the `deleteAccount` with password version. What should we do? Remove for now I guess